packages/babel-plugin-transform-classes/src/vanilla.js

```diff
@@ -522,9 +522,18 @@ export default class ClassTransformer {
       if (this._processMethod(node, scope)) return;
     }
 
+    // Check if it is a static get or set
+    if (node.static && (node.kind === "get" || node.kind === "set")) {
```

Comment:
well, this is more optimized than before, dunno how much do you want to spend on this, but in general i think this should be more optimized even further - get+set pair for the same property could still produce a single `Object.defineProperty` call, while at the moment this will still fallback to using `createClass` helper

Reply:
It will be issues with computed properties like you mentioned here : https://babeljs.slack.com/archives/C062RC35M/p1515617623000644?thread_ts=1515589048.000272&cid=C062RC35M
 Should I optimise this PR in any way? 🙂

Reply:
You can optimize it if the getter and setter name are not computed.

Reply:
done the optimisation but I am blocked by : https://github.com/babel/babel/pull/7274 : In case I am doing it wrong, please guide me 🙂